matlab/CMakeLists.txt

```diff
@@ -61,7 +61,8 @@ endif()
 # ignoring the non-concrete types (type aliases)
 set(ignore
     gtsam::Point2
-    gtsam::Point3)
+    gtsam::Point3
+    gtsam::CustomFactor)
```

Comment:
Can you add a comment in gtsam.i that CustomFactor won't be wrapped for matlab?

Reply:
done